### DISCUSSION TITLE
Why is my Vite so slow?

### DISCUSSION THREAD

**Discussion by gustavotoyota** [quasarframework/quasar#14218] [CLI - SSR mode]
Looking at the Network tab I see that each file takes around 8 seconds loading with the (pending) status.
Sometimes Chrome shows yellow warnings about source map loading timeout, so I tried disabling source maps on Chrome but it didn't help.
I think Vite is probably waiting for something and reaches a timeout and then finally loads my scripts, but I couldn't find out what it is or how to prevent it from waiting.

My `quasar info`:
```
Operating System - Windows_NT(10.0.19044) - win32/x64
NodeJs - 16.13.2

Global packages
  NPM - 8.4.0
  yarn - 1.22.18
  @quasar/cli - 1.3.2
  @quasar/icongenie - 2.5.3
  cordova - Not installed

Important local packages
  quasar - 2.7.7 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.0.6 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.15.1 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.37 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.1.3
  pinia - 2.0.18 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  vite - 2.9.15 -- Native-ESM powered web dev build tool
  eslint - 8.22.0 -- An AST-based pattern checker for JavaScript.
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - Not installed
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - DESKTOP-XXXXXXX
  Wi-Fi - 192.168.1.4
```

**Comment by MilosPaunovic** [comment]
It might be caused by something else in your code, not strictly Quasar related.

My suggestion would be to create a minimal reproduction repository and try to pinpoint the problem. If indeed turns out that it's Quasar related, provide us with a reproduction and someone will take a look at it.

**Reply by gustavotoyota** [reply]
How can I see what Vite is doing?
I tried `extendViteConf(viteConf) { viteConf.logLevel = 'info'; }`, but it doesn't show the logs.

**Reply by MilosPaunovic** [reply]
You are on the right track with `extendViteConf`:
https://quasar.dev/quasar-cli-vite/handling-vite#usage-with-quasar-config-js

But bare in mind that `info` is the default log level.
https://vitejs.dev/config/shared-options.html#loglevel

**Reply by gustavotoyota** [reply]
I change the logLevel but still can't see the Vite logs. How can I see what Vite is doing through quasar?

**Comment by gaoyusongcn** [comment]
I have this problem too.  that veryyyyyyy slow... 

I use the default template test !!!!! what's happen

**Comment by pdanpdan** [comment]
Most probably you are using your company provided machine and they updated one or more of their agents (most probably clown strike) to a version that is even worse the usual.

**Reply by gaoyusongcn** [reply]
@pdanpdan  

But I did not have this problem when I tested the vite of my own project. 

This problem sometimes arises in Quasar and I just use the default template `npm init quasar`
